src/main/io/rc_controls.c

```diff
@@ -342,7 +343,16 @@ void updateActivatedModes(modeActivationCondition_t *modeActivationConditions)
         modeActivationCondition_t *modeActivationCondition = &modeActivationConditions[index];
 
         if (isRangeActive(modeActivationCondition->auxChannelIndex, &modeActivationCondition->range)) {
```

Comment:
maybe low-pass filtered value will work (averaging is one kind of LPF).
Filtering will introduce delay. But acceptance of new value is delayed anyway (signal must be delayed to check if it is stable)
And there should be some method to estimate windowed signal variance using filter. Maybe just low-pass filter and filtered square difference would do the trick ...